### PR TITLE
Fix failing to copy upgrade test result xml following test server con…

### PR DESCRIPTION
**Pull request by seamuslee001**
…fig change

@totten I have deployed this for the DFL build on test-1 and running a manual test now

### CODE REVIEW

**Comment by seamuslee001** [comment]
I'm going to merge this in so I can run this on test-1 where it is showing good promise

File: bin/civi-test-run [hunk 1]
```diff
@@ -118,7 +120,11 @@ function task_upgrade() {
     EXITCODES="$EXITCODES upgrade-test"
   fi
   echo "Found EXITCODES=\"$EXITCODES\""
-  $GUARD cp "$PRJDIR/app/debug/$BLDNAME/civicrm-upgrade-test.xml" "$JUNITDIR/"
+  if [ -d "$PRJDIR/build/$BLDNAME/.civibuild" ]; then
```
Comment (totten) [inline bin/civi-test-run:123]: Yup, this looks sensible.

File: .loco/config/mysql-common/my.cnf.php [hunk 2]
```diff
@@ -37,6 +37,7 @@ function matchVer($pat) {return (bool) preg_match($pat, ver());}
 <?php if (matchVer('/Ver 5.6/')) { ?>
 innodb_large_prefix = 1
 <?php } ?>
+<?php if (!matchVer('/Ver 8.0/')) { ?>innodb_file_format = Barracuda<?php } ?>
```
Comment (seamuslee001) [inline .loco/config/mysql-common/my.cnf.php:40]: @totten this was to fix some Min test failures I have no idea how to actually apply this so found the relevant files and manually fixed them on the server

Comment (totten) [inline .loco/config/mysql-common/my.cnf.php:40]: I like how much easier that is now. :)

Small revision: https://github.com/civicrm/civicrm-buildkit/pull/528

Comment (totten) [reply .loco/config/mysql-common/my.cnf.php:40]: Good q. Easiest ways to apply this:

* For a local environment (where one uses `loco run`), just add the `-f` / `--force` option (i.e. `loco run -f`).
* For CI environment (where there are several systemd services), use `nix/bin/reset-ci.sh`.

In both cases, it'll re-generate the concrete config files and start the services.

(There are also more fine-tuned options like `loco init -f mysql`.)

**Pull request merged**